### PR TITLE
Add features for `Account`

### PR DESCRIPTION
**Pull request by kriii**
Changes:
- Support mnemonic
- Separate `Account` and `Wallet`
- Support creating `Account` from entropy

I have a curiosity about function [concatHexPrefixed](https://github.com/ainblockchain/ain-util/blob/master/src/index.ts#L839-L841). Why does it slice `b`?

And some notes about mnemonic:
It seems there are two methods to support mnemonic, and they have cons resp.
1. use `bip-utils` - I had many failures to install because this required installing `rust` and doing some settings. But it was for the installing subpackage meaningless about mnemonic.
2. use `mnemonic` and `bip32` - Easy to install, but `bip32` doesn't support type hinting, so `mypy` hates it.

The chosen option is the second with ignored the type hint.

### CODE REVIEW

File: ain/account.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    @classmethod
+    def fromMnemonic(cls, mnemonic: str, index: int = 0):
+        """
+        Returns an Account with the given mnemonic.
+        """
+        return cls(mnemonicToPrivatekey(mnemonic, index))
+
+    @classmethod
+    def fromEntropy(cls, entropy: str = None):
```
Comment (liayoo) [inline ain/account.py:35]: suggestion) how about `create()`, since `fromMnemonic()` is getting an account from a mnemonic but this function creates a new account?

**Comment by liayoo** [comment]
As per your question on `concatHexPrefixed()`, I think it was meant to slice the hex prefix '0x', but I do agree that the function should check if b is hex prefixed first..! Let me fix that. Thanks!

File: ain/account.py (hunk 1 above)
Comment (kriii) [reply ain/account.py]: Changed! PTAL.

File: tests/test_ain_account.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+)
+
+class TestAccount(TestCase):
+    def testAccountFromMnemonic(self):
+        account = Account.fromMnemonic(mnemonic)
+        self.assertEqual(account.private_key, mnemonicPrivateKey.hex())
+        self.assertEqual(account.public_key, mnemonicPublicKey.hex())
+        self.assertEqual(account.address, mnemonicAddress)
+    
+    def testAccountCreateFromEntropyNone(self):
```
Comment (liayoo) [inline tests/test_ain_account.py:18]: could you add a test case with entropy != None?

Comment (kriii) [reply tests/test_ain_account.py:18]: Added!

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

**Comment by kriii** [comment]
Merged! Thanks for the review!
